printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -15,101 +15,372 @@ The objectives for this project are:
+
+You need to set the probe offset within your `printer.cfg`  
 
 There is now an arrow on the probe telling you where should the switch pole be to have the correct offset.
 The probe offsets are:
-- z_offset = 6.42
-- x_offset: 0
-- y_offset: 19.75
+```python
+z_offset = 6.42
+x_offset: 0 
+y_offset: 19.75
+```
+
+### Z endstop and Probe configuration
+
+If you want to use the Klicky Probe as your Z endstop, you need to change the `endstop_pin:` under the `[stepper_z]` section to `probe:z_virtual_endstop` . Just comment out the old one and add a new line `endstop_pin: probe:z_virtual_endstop`. 
+
+You don't need to change anything else in the printer.cfg, so leave it as it is, you can easily revert back to the original setup.
+
+### Use Klicky Probe with/without Z endstop switch (Voron)
+
+If you want to use the Z endstop switch of the Voron you also need to set the following two lines, this is the Z endstop Location from your `printer.cfg`.
+
+```python
+variable_z_endstop_x:
+variable_z_endstop_y:
+```
+
+If you want to use your Klicky Probe as a Z endstop, then you need to set the two lines to, `0`.
+
+```python
+variable_z_endstop_x:     0
+variable_z_endstop_y:     0
+```
+
+### Adjust Probe Pickup Position
+
+One of the last things we need to do is to adjust the probe pickup position.
+
+For this we need to make sure that the gantry is [deracked](https://www.youtube.com/watch?v=cOn6u9kXvy0), the x and y axis are homed and the probe is manually attached to the AB-Mount.
+
+Now manually move the toolhead to the probe dock and move it so far to the back that the probe docks, note the Y-Position.
+Next, again manually, move the toolhead left and right until the probe it is perfectly aligned with the mount, note the X.Position.
+
+Open you `Klicky Probe.cfg` and find the `#dock location` section and edit the following two line
+
+```python
+variable_docklocation_x:
+variable_docklocation_y:
+```
+
+If you have your Dock mounted to the bed then you need to adjust the `variable_docklocation_z:`, too.
+
+#### Hall sensors as Y endstop
+
+If you are using a hall sensor as endstop, you need to make sure that on your Y maximum, the gantry is almost hitting the AB motor mounts, you can have a Y position maximum "behind" the Y endstop position, like this:
+```python
+[stepper_y]
+position_endstop: 303
+position_max: 305
+```
+
+### Automatic Z Calibration
+
+If you want to use the Z endstop switch of the Voron to calculate the Z-Offset, use the new [automatic Z calibration](https://github.com/protoloft/klipper_z_calibration).
+
+Most of necessary macros are already included in the klick-probe.cfg, what is missing is the specific z_calibration configuration and the macro that is called to do the actual calibration.
+All of this is included in the [Klicky automatic Z calibration configuration](https://github.com/jlas1/Klicky Probe/blob/main/klipper_macro/z_calibration.cfg)
```

Comment:
Broken link because of the space in the URL, instead of `Klicky Probe` should be `Klicky-Probe`

Reply:
done, also moved that to a reference, as that file is also in this repository.